README.md

```diff
@@ -18,210 +18,181 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
-
-### Database
-Darkflame Universe utilizes a MySQL/MariaDB database for account and character information.
-
-Initial setup can vary drastically based on which operating system or distribution you are running; there are instructions out there for most setups, follow those and come back here when you have a database up and running.
-
-* All that you need to do is create a database to connect to.  As long as the server can connect to the database, the schema will always be kept up to date when you start the server.
+For Windows, run the following command in the `Command Prompt (MariaDB xx.xx)` terminal
+```bash
+# Logs you into the mysql instance
+mysql -u root -p
+# You will then be prompted for the password you set for root during installation of MariaDB
+```
 
-#### Configuration
+Now that you are logged in, run the following commands.
 
-After the server has been built there should be four `ini` files in the build director: `sharedconfig.ini`, `authconfig.ini`, `chatconfig.ini`, `masterconfig.ini`, and `worldconfig.ini`. Go through them and fill in the database credentials and configure other settings if necessary.
+```bash
+# Creates a user for this computer which uses a password and grant said user all privileges.
+# Change mydarkflameuser to a custom username and password to a custom password. 
+GRANT ALL ON *.* TO 'mydarkflameuser'@'localhost' IDENTIFIED BY 'password' WITH GRANT OPTION;
+FLUSH PRIVILEGES;
 
-#### Migrations
+# Then create a database for Darkflame Universe to use.
+CREATE DATABASE darkflame;
+```
 
-The database is automatically setup and migrated to what it should look like for the latest commit whenever you start the server.
+## Build the server
+You can either run `build.sh` when in the root folder of the repository:
 
-#### Verify
+```bash
+./build.sh
+```
 
+Or manually run the commands used in [build.sh](build.sh).
+
+### Notes
+Depending on your operating system, you may need to adjust some pre-processor defines in [CMakeVariables.txt](./CMakeVariables.txt) before building:
+* If you are on MacOS, ensure OPENSSL_ROOT_DIR is pointing to the openssl root directory.
+* If you are using a Darkflame Universe client, ensure NET_VERSION is changed to 171023.
+
+## Configuring your server
+This server has a few steps that need to be taken to configure the server for your use case.
+
+### Required Configuration
+Darkflame Universe can run with either a packed or an unpacked client.
+Navigate to `build/sharedconfig.ini` and fill in the following fields:
+* `mysql_host` (This is the IP address or hostname of your MariaDB server.  This is highly likely `localhost`)
+* `mysql_database` (This is the database you created for the server)
+* `mysql_username` (This is the user you created for the server)
+* `mysql_password` (This is the password for the user you created for the server)
+* `client_location` (This is the location of the client files.  This should be the folder path of a packed or unpacked client)
```

Comment:
Possibly make a note that the path to the client can't contain spaces. I had issues with the client folder name having a space on Windows.

Reply:
NB: When I tried (on Linux), I don't think spaces caused an issue. I didn't test thoroughly though